x-pack/legacy/plugins/canvas/public/components/editor/editor.tsx

```diff
@@ -0,0 +1,158 @@
+/*
```

Comment:
Thanks for the great work on this! Keen to use this component in Console ASAP :)